modules/release-notes/pages/relnotes.adoc

```diff
@@ -5,6 +5,254 @@ It also includes enhancements to the Eventing and Search services.
+
+| https://issues.couchbase.com/browse/MB-33025[MB-33025^]
+| *Summary*: Fixed an issue where the analytics HTTP server incorrectly assumed that all request payloads are UTF-8 encoded. The analytics HTTP responses are encoded using system-default encoding and the service will now attempt to honor any requested encodings (Accept-Charset), or default to UTF-8.
+
+| https://issues.couchbase.com/browse/MB-33016[MB-33016^]
+| *Summary*: Fixed an internal compiler error caused by index selection when the search expression was not constant.
+
+| https://issues.couchbase.com/browse/MB-32777[MB-32777^]
+| *Summary*: Fixed an issue where array functions, most notably the array_intersect() function, caused an out of memory exception due to object creations, especially for large datasets.
+
+| https://issues.couchbase.com/browse/MB-32775[MB-32775^]
+| *Summary*: Fixed an issue to catch all unexpected exceptions on network threads and close the connection in such cases to avoid attempting to process the message causing the exception forever.
+
+| https://issues.couchbase.com/browse/MB-32745[MB-32745^]
+| *Summary*: The analytics service now uses the Couchbase temp directory, which is set to `/var/lib/couchbase/tmp` by default, and no longer requires access to the operating system's temp directory. 
+|===
+
+==== Cross Datacenter Replication (XDCR)
+
+[#table_fixedissues_v602-xdcr,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-33970[MB-33970^]
+| *Summary*: By default, XDCR will prevent replications from being created on a source ephemeral bucket with NRU policy. This release of Couchbase Server introduces a special XDCR internal settings flag, `AllowSourceNRUCreation`, that allows replications to be created on a source ephemeral bucket with NRU policy. 
+
+----
+curl -X POST -u Administrator:password http://127.0.0.1:9998/xdcr/internalSettings -d AllowSourceNRUCreation="true"
+----
+
+WARNING: Changing the internal replication setting will cause all XDCR processes to restart. This causes temporary disruption as all existing replications will have to restart.
+
+| https://issues.couchbase.com/browse/MB-33671[MB-33671^]
+| *Summary*: Following an issue on a remote cluster, XDCR failed to recover and threw an exception.
+
+| https://issues.couchbase.com/browse/MB-33582[MB-33582^]
+| *Summary*: After upgrading to 6.0.x, it was observed that the compression type not set automatically. This has been fixed.
+
+| https://issues.couchbase.com/browse/MB-32814[MB-32814^]
+| *Summary*: Fixed REST endpoint error handling for settings errors and nil value types.
+|===
+
+==== Data Service
+
+[#table_fixedissues_v602-kv-data,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-34507[MB-34507^]
+| *Summary*: Fixed an issue observed when attempting to modify via subdoc by reseting the datatype to XATTRs if the pre-expiry callback returns a non-empty response.
```

Comment:
Suggest something along the lines of:

*Summary*: Fixed an issue where documents with a TTL, containing system XATTRs, and which are compressed, may have their `datatype` metadata incorrectly set. This prevents any subsequent modification of the XATTRs on the deleted document, until the document is re-created.